Dockerfile

```diff
@@ -23,5 +23,6 @@ ARG TARGETARCH
 
 FROM gcr.io/distroless/static:latest
 COPY --from=builder /src/linstor-scheduler-extender /linstor-scheduler-extender
+COPY --from=builder /src/linstor-scheduler-admission /linstor-scheduler-admission
```

Comment:
Looks like the `go build` command needs to be updated, otherwise it doesn't produce the binaries here.

I needed to add a `-o .` to the flags

Reply:
Thank you, the command is fixed.
Any way I was considering to make separate image for linstor-scheduler-admission. What do you think?

Reply:
I'm fine with how it is now, but if you think it's cleaner to have a separate image, I'm fine with that too.

Reply:
We're always run admission webhooks on master nodes to minimize impact of workloads which affects api interaction directly.
Otherwise I can see that nginx-ingress runs webhook aside the controller, thus let it be as is.